Shopify/hydrogen
# Applied gift card

### WHY are these changes introduced?

* Add the newly introduced `cartGiftCardCodesUpdate` mutation to cartCreateHandler
* Update skeleton template to handle gift card codes

### WHAT is this pull request doing?





### HOW to test your changes?

1. Go to admin and create a couple gift cards, these code are one-time view only so keep a copy it of it.
2. Run skeleton template and add an item to cart
3. Add gift card codes
4. Go to checkout and see they are applied

#### Post-merge steps



#### Checklist
